saturn_client/core.py

```diff
@@ -455,4 +439,10 @@ def _maybe_name(_id):
     """Return message if len of id does not match expectation (32)"""
     if len(_id) == 32:
         return ""
-    return " Maybe you used name rather than id?"
+    return "Maybe you used name rather than id?"
+
+
+def _http_error(response: requests.Response, resource_id: str):
+    """Return HTTPError from response for a resource"""
+    response_message = response.json().get("message", "")
```

Comment:
sorry, didn't notice this on my first review. I think we should have some actual text here, even if it's generic.

```suggestion
    response_message = response.json().get("message", "saturn-client encountered an unexpected error.")
```

I don't think it would be a fun experience to see this in your logs:

> [FATAL]    Maybe you used name rather than id?

Reply:
Yeah good call